crypto/crmf/crmf_local.h

```diff
@@ -329,7 +330,7 @@ struct ossl_crmf_certtemplate_st {
 struct ossl_crmf_certrequest_st {
     ASN1_INTEGER *certReqId;
     OSSL_CRMF_CERTTEMPLATE *certTemplate;
-    STACK_OF(OSSL_CRMF_ATTRIBUTETYPEANDVALUE) *controls;
+    STACK_OF(OSSL_CRMF_ATTRIBUTETYPEANDVALUE /* Controls expanded */) *controls;
```

Comment:
Not sure about style here.

Reply:
At least `check-format.pl crypto/crmf/crmf_local.h` is happy with it.